include/vrv/options.h

```diff
@@ -496,6 +496,9 @@ class Options {
     OptionBool m_svgHtml5;
     OptionInt m_unit;
     OptionBool m_useFacsimile;
+    OptionBool m_createDefaultSyl;
+    OptionBool m_createDefaultSylBBox;
+    OptionBool m_useRotate;
```

Comment:
The second one creates zones for syls that don't have one. Do you have an alternative in mind to the options? We only put them here so they were happening by default.

Reply:
Are they cases where you turn them off? And cases where you turn only one for them on? Because both are very specific to neumes, so I am questioning the need for having options. Do we need them at all? If yes, do we need two? We can leave them but we would need better names.
For the first one, I don't like the fact the MEIInput changes the content of the tree while reading. In other cases, this is happening after the document is loaded. See methods such as Doc::ConvertMarkupDoc. It would be better to add something for syllables there (a bitfield MARKUP_SYLLABLE_CONTENT) and if possible fix both empty syllables and facsimile zones in one functor. Not sure it can work for facsimile zone, though. What do you think? It is a bit over the top to have an additional functor for this but it would be clearer.

Reply:
Not where we do with Neon, but if there was hypothetically a neume editor that didn't work with facsimiles then it would make sense to create default `syl` elements for editing but not create a `zone`. With `m_useRotate`, that just be wrapped into the facsimile option since we wrote this when `@rotate` was not standard.

I'd be fine with adding a functor to handle the syllables and zones outside of `MEIInput`. Do you think it makes sense to keep the two syl options separate? Or at all. It could just add a `syl` element since a syllable should have text and only add a zone if the facsimile option is passed.

Reply:
I think it is fine not to have the options at all if this works for you. That is, making sure there is a `<syl>` in `<syllable>` in all cases. You can add a `<zone>` only if the Doc is of type facs. This is acceptable. Comment the code as much as necessary to make all this clear.

(For the future: it would be best to avoid addition of content in Verovio, for various reasons. ConvertMarkup goal is primarily to convert to another representation - e.g., attribute into element. I would prefer it not to become a place where we "fix" the encoding")

Reply:
Got it! And sorry about including that kind of feature here.